flow/record/fieldtypes/__init__.py

```diff
@@ -32,6 +40,27 @@
 path_type = pathlib.PurePath
 
 
+def flow_record_tz(default_tz: str = "UTC") -> Optional[ZoneInfo]:
+    """Return a ZoneInfo object based on the ``FLOW_RECORD_TZ`` environment variable.
+
+    Args:
+        default_tz: default timezone if ``FLOW_RECORD_TZ`` is not set (default: UTC)
+
+    Returns:
+        None if ``FLOW_RECORD_TZ=NONE`` otherwise ``ZoneInfo(FLOW_RECORD_TZ)``
+    """
+    tz = os.environ.get("FLOW_RECORD_TZ", default_tz)
+    if tz.upper() == "NONE":
+        return None
+    return ZoneInfo(tz)
```

Comment:
Perhaps throw a try/except on here for `ZoneInfoNotFoundError`? Or would you want it to intentionally throw an error?

Might be useful to show a warning and then default to UTC (straight from `datetime.timezone.utc` to prevent further errors with `ZoneInfo`)? On Windows this will currently always error if you're missing the `tzdata` package.

Reply:
Showing a warning and falling back to UTC is a good idea, will implement this.

Reply:
Showing a warning and then falling back to UTC is a good idea. will implement this.